package.json

```diff
@@ -14,5 +20,12 @@
     "rimraf": "~2.2.8",
     "vinyl-source-stream": "^1.0.0",
     "virtual-dom": "^1.3.0"
+  },
+  "main": "src/index.coffee",
+  "browserify": {
+    "transform": [
```

Comment:
Since we invoke browserify programmatically instead of a build step, it seems more modular to put these options into the invocation code. Also, are we using `brfs` yet?